packages/flame/lib/src/parallax.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:async';
 import 'dart:ui';
 
+import 'package:collection/collection.dart';
```

Comment:
is this a new dependency? I am weary of adding deps for `flame` itself just for sugars, as flame itself is a library used by other people. also if we were to I would prefer to add one, I would prefer dartlin :p

Reply:
No, this is part of dart. It's weird that it is not part of the core collection lib, but I think it has something to do with the null-safe transition. There are all kinds of great extension collection methods in there.

Reply:
Ah, it's part of Flutter, not dart.

Reply:
oh wow, that is really useful to know, I had no idea that was available. cool stuff!